spec/support/step_nav_steps.rb

```diff
@@ -212,6 +212,10 @@ def then_I_can_see_a_success_message(message)
 
   alias_method :and_I_can_see_a_success_message, :then_I_can_see_a_success_message
 
+  def and_I_am_the_reviewer
```

Comment:
There is a `and_I_am_the_reviewer` method in `spec/features/step_by_step_actions_spec.rb `.  Could this (and `and_I_am_the_step_by_step_author`) be removed from `spec/features/step_by_step_actions_spec.rb` and replaced by methods in this support so there's no duplication?